src/assets/translations/cs-CS.json

```diff
@@ -2561,6 +2561,7 @@
       "Provisioning Criteria": "Kritéria poskytování",
       "Provisioning Entries": "Poskytování záznamů",
       "Provisioning criteria definitions": "Vyplňte všechny definice kritérií poskytování.",
+      "Recover From Guarantor": "Zotavit se z ručitele",
```

Comment:
Please send a separate PR for this or involve in the description and PR title this change as well!